### DISCUSSION TITLE
Event Stream Person Detection

### DISCUSSION THREAD

**Discussion by broyuken** [tsightler/ring-mqtt#404] [Feature Requests]
Hello, 

Could I request the ability to add an option in the event stream viewer to view the last 5 motion events where a person is detected instead of just the last 5 motion events?

**Comment by tsightler** [comment]
You can request it, but I'm not sure of when I will get to it!  :-)

Unfortunately I have an extremely limited amount of time over the next couple of months, although this is actually a pretty easy request and maybe I'll somehow find a few minutes to sneak it in, especially since even I have wanted this feature sometimes!  I also want to provide the ability to choose transcoded videos rather than the raw video, which I think will provide the pre-roll recordings as well, although I haven't really tested this yet.

**Comment by broyuken** [comment]
There is no expectation for a quick turnaround for this or anything that is requested to be done for free in your spare time. If you have the time to develop it, great, if not I hope it at least is somewhere in your brain for when you do have the time/desire to add things.

**Comment by dmkjr** [comment]
I second this.  I see all kinds of integrations this request being made.  Thanks for your work @tsightler
